trestle/utils/fs.py

```diff
@@ -539,3 +540,21 @@ def save_top_level_model(
     if not full_model_path.parent.exists():
         full_model_path.parent.mkdir(parents=True, exist_ok=True)
     model.oscal_write(full_model_path)
+
+
+def merge_dicts_deep(a: Dict[Any, Any], b: Dict[Any, Any]) -> None:
```

Comment:
Im not sure if this belongs here as `fs` is a "common file system utilities" and this function deals with dictionary.

Reply:
I moved it to where it is used in control_io.py and made it static so it can be used by others if need be.